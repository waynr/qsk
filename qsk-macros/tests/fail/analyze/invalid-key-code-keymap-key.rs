use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer[Meow]: {
            MEOW -> F,
        },
    );
}
