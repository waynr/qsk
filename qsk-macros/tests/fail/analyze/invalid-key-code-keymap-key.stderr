error: invalid key code when converting parse::Key to KeyCode: VariantNotFound
 --> tests/fail/analyze/invalid-key-code-keymap-key.rs:6:13
  |
6 |             MEOW -> F,
  |             ^^^^
